changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
+      "[New] Merge button in branch selector - #5929. Thanks @bruncun!",
+      "[New] Merge conflict resolution flow - #5400",
+      "[Fixed] Single file commit summary now matches behavior of GitHub.com - #6017. Thanks @Daniel-McCarthy!",
```

I think this should be improved, not fixed, right. It's not a bug I don't think.

```suggestion
      "[Improved] Default commit summary more explanatory and consistent with GitHub.com - #6017. Thanks @Daniel-McCarthy!",
```